math/cos_test.go

```diff
@@ -5,7 +5,7 @@ import (
 	"testing"
 )
 
-const epsilon = 0.001
+const epsilonCos = 0.001
```

Comment:
Why different? Doesn't this mean that this is not a good algorithm that it changes the precision?

Reply:
I'm sorry, this is a poorly written test by me

Reply:
Fix